src/wffile.c

```diff
@@ -131,9 +131,18 @@ DWORD MKDir(
    else
    {
       dwErr = GetLastError();
+
+      // Unfortunatley CreateDirectoryEx does not support developer mode, so we have to create symbolic ourselves
+      if (ERROR_PRIVILEGE_NOT_HELD == dwErr && (GetFileAttributes(pSrc) & FILE_ATTRIBUTE_REPARSE_POINT))

```

Comment:
I think this should check that it's a symbolic link reparse tag.  I know that seems gross, but I think it's needed anyway, because....